codespell_lib/data/dictionary.txt

```diff
@@ -43103,8 +43107,11 @@ queing->queueing
 queires->queries
 queiried->queried
 queisce->quiesce
+quereis->queries
+queres->queries, queers,
 queriable->queryable
 quering->querying
+queris->queries
```

Comment:
Perhaps `querist`, although probably rare?

Reply:
`querist` is too rare.